docs/start/integration.md

```diff
@@ -406,44 +414,44 @@ The simplest way to decode a serialized raw transaction is using the [`Transacti
 
 - SingleSig spawn: template `sm1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqg56ypy7`, method `0` (or unset)
 - SingleSig spend: template `sm1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqg56ypy7`, method `16`
 - MultiSig spawn: template `sm1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqsl0g40s`, method `0` (or unset)
 - MultiSig spend: template `sm1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqsl0g40s`, method `16`
 
-Spend transactions have a destination (recipient) account and an amount. These are encoded inside the transaction's raw payload. These data are not currently exposed via the API; we're [working on it](https://github.com/spacemeshos/api/issues/288). For now, in order to fully parse a transaction, you'll have to manually write code such as this [VM code](https://github.com/spacemeshos/go-spacemesh/blob/3d8eedfd44c6f9f8987e604b768596f6001fe703/genvm/templates/wallet/handler.go#L57-L72) that handles tx parsing.
+Spend transactions have a destination (recipient) account and an amount. These are encoded inside the transaction's raw payload. This data is not currently exposed via the API but we're [working on it](https://github.com/spacemeshos/api/issues/288). For now, in order to fully parse a transaction, you will have to manually write code such as this [VM code](https://github.com/spacemeshos/go-spacemesh/blob/3d8eedfd44c6f9f8987e604b768596f6001fe703/genvm/templates/wallet/handler.go#L57-L72) that handles transaction parsing.
 
-Note: there is one additional template, a Vault wallet, that's a special type of MultiSig wallet that's used to access the funds locked in the genesis vaults. However, vaulted funds don't begin to vest until one year post-genesis, i.e., in July 2024. Vault transactions shouldn't appear before that time, and will be invalid before that time.
+Note: there is one additional template: a Vault wallet, this is a special type of multisig wallet that is used to access the funds locked in the genesis vaults. However, vaulted funds do not begin to vest until one year post-genesis (i.e., in July 2024). Vault transactions should not appear before that time and will be invalid if they do so
 
 ### Lifecycle
 
 The process of creating, broadcasting, and tracking a transaction looks something like the following:
 
-1. Calculate the recipient account address, following the instructions above
-1. Construct a transaction, setting principal and recipient, amount, gasprice, etc.
-1. Sign the transaction using the principal's private key
-1. Send the transaction to a full node
-1. The full node gossips the transaction to the network
-1. Within a few minutes, the transaction should get included in one or more proposals, then mined into a block
-1. Within a few minutes the layer gets approved, and the transaction gets applied
-1. Check the transaction status with a full node and see that it was applied
+1. Calculate the recipient account address, following the instructions above.
```

Comment:
Please don't renumber lists. It's a feature of markdown that it performs the numbering for you automatically, which makes it much easier to update the list in the future.

Reply:
Did not know that! Reverted back to `1.`.